app/uk/gov/hmrc/servicedependencies/connector/GithubRawConnector.scala

```diff
@@ -0,0 +1,58 @@
+/*
+ * Copyright 2020 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package uk.gov.hmrc.servicedependencies.connector
+
+import cats.implicits._
+import javax.inject.{Inject, Singleton}
+import org.yaml.snakeyaml.Yaml
+import uk.gov.hmrc.http.{HeaderCarrier, HttpClient, HttpReads, HttpResponse, UpstreamErrorResponse}
+import uk.gov.hmrc.servicedependencies.config.ServiceDependenciesConfig
+import HttpReads.Implicits._
+
+import scala.collection.JavaConverters._
+import scala.concurrent.{ExecutionContext, Future}
+import scala.util.Try
+
+@Singleton
+class GithubRawConnector @Inject()(
+  httpClient               : HttpClient
+, serviceDependenciesConfig: ServiceDependenciesConfig
+)(implicit
+  ec: ExecutionContext
+) {
+  def decomissionedServices(implicit hc: HeaderCarrier): Future[List[String]] = {
+    val url = s"${serviceDependenciesConfig.githubRawUrl}/hmrc/deploy-with-docktor/master/decommissioning/microservices-to-decommission.yaml"
```

Comment:
would it be worth having this url configurable? we dont own the repo so its possible it could change if they reorganize stuff?

Reply:
although this would make it difficult to stub in the catalogue-data-stub

Reply:
:+1: